Mathlib/Topology/Algebra/Module/LocallyConvex.lean

```diff
@@ -192,3 +194,45 @@ instance Prod.locallyConvexSpace [TopologicalSpace E] [TopologicalSpace F] [Loca
     (locallyConvexSpace_induced (LinearMap.snd _ _ _))
 
 end LatticeOps
+
+section LinearOrderedSemiring
+
+instance {R : Type*} [TopologicalSpace R] [LinearOrderedSemiring R] [OrderTopology R] :
```

Comment:
Perhaps
```suggestion
instance LinearOrderedSemiring.toLocallyConvexSpace {R : Type*} [TopologicalSpace R] [LinearOrderedSemiring R] [OrderTopology R] :
```
so that this is referenceable.